### PR TITLE
Issue 76 config bind and port

### PR DESCRIPTION
**Pull request by JivanAmara**
Rename current (string) 'port' config parameter to 'bind'.
Add int 'port' parameter (used in generating urls).

### CODE REVIEW

**Comment by ARolek** [comment]
The v0.6.0 branch no longer exists. moving this to be based of the new v0.7.0 branch.

**Comment by JivanAmara** [comment]
@ARolek Did this not make it into v0.6.0?

**Comment by ARolek** [comment]
@JivanAmara I don't think it did. I will get the change into v0.7.0

File: config/config.go [hunk 1]
```diff
@@ -35,9 +35,12 @@ type Config struct {
 }
 
 type Webserver struct {
+	Bind              string `toml:"bind"`
 	HostName          string `toml:"hostname"`
-	Port              string `toml:"port"`
+	Port              int    `toml:"port"`
```
Comment (ear7h) [inline config/config.go]: With Bind do we still need port and hostname?

**Comment by ARolek** [comment]
After talking with @gdey we're going to change this behavior to:

- the `port` config will be converted from a string to a uint
- a new `bind` value will be added which will accept a string in the format of the current `port` config val.
- if both `port` and `bind` are set we throw an error during config validation.

**Comment by gdey** [comment]
Looks like we found a way forward. But this never got accepted. Do we want to continue with this.

The way forward looks to be this:

`port` is going to be a uint
`bind` is a string.
both options are mutually exclusive

**Comment by gdey** [comment]
Looking at the code, it may make sense to close this PR, open and issue and create a fresh patch. Code has change sigificantly since this PR was opened.

**Comment by ARolek** [comment]
@gdey I think we should too. This is not a priority, so I think we close this PR for now and revist at a later date.

**Pull request closed without merging**